model/__init__.py

```diff
@@ -782,3 +782,7 @@ def set_certificate(self, certificate: str):
 
     def _required_attributes(self):
         return ['private_key', 'certificate']
+
```

Comment:
usually (I know, we do not yet have done this properly everywhere in our code-base, yet), there are two blank lines between class definitions

Reply:
done